dashboard/app/controllers/plc/course_unit_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Plc::CourseUnitController < ApplicationController
+  # PUT /launch/:course_unit_name
+  def launch
```

Comment:
Do we have some sort of restriction (ie, cancan) on who can launch a course?

Reply:
I think `find_by!` will automatically raise this error. But, I kind of like the more explicit raise here actually, more readable.

Reply:
Added a restrict to require levelbuilder mode for launch